basic-assembly/basic-maven-plugin/pom.xml

```diff
@@ -14,11 +14,11 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.eclipse.sw360.antenna</groupId>
-        <artifactId>antenna-basic-assembly</artifactId>
+        <artifactId>basic-assembly</artifactId>
         <version>${revision}${qualifier}</version>
     </parent>
 
-    <artifactId>antenna-maven-plugin</artifactId>
+    <artifactId>basic-maven-plugin</artifactId>
 
```

Comment:
And the same here, from Antenna perspective this is THE Maven Plugin, not a simple not really usable one. No one has to adapt this, if he only wants to use Antenna, right?